### PR TITLE
Integrated player testcase

### PR DESCRIPTION
**Pull request by Nandana-NNR**

### CODE REVIEW

File: cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+        Examples:
+            | Scenario                    | API_Key                              | Validation_key                   | method         |
+            | Validate create api         | create broadcastPlayer instance      | broadcastPlayer instance         | create         |
+            | Validate status api         | get status of broadcastPlayer        | status of broadcastPlayer        | status         |
+            | Validate provideStatus api  | get broadcast status of player       | broadcast status of player       | provideStatus  |
+            | Validate createResponse api | get response from createprovider     | response of createprovider       | createResponse |
+            | Validate createError api    | get error from createprovider        | error of createprovider          | createError    |
+            | Validate statusResponse api | get response from statusprovider     | response of statusprovider       | statusResponse |
+            | Validate statusError api    | get error from statusprovider        | error of statusprovider          | statusError    |
```
Comment (neeradanelxsi) [inline cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature:36]: not sure I understand these test cases . Are you trying to call rpc-only methods here ? 


PlayerApp creates the Player based on the first IUI request ... Based on the type either Broadcast or Streaming Player gets created...
Then next call from IUI would be load/play content which will be performed in the Player created by above step

File: cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature [hunk 2]
```diff
@@ -0,0 +1,64 @@
+Feature: IntegratedPlayer
+
+    @initialization
+    Scenario: Launch FCA for 'IntegratedPlayer'
+        Given the environment has been set up for 'IntegratedPlayer' tests
+        And 3rd party 'certification' app is launched with 'inactive' state
```
Comment (neeradanelxsi) [inline cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature:6]: we need the test case to be separate here. 
Creating player can be a single test case
Then all the player method calls like load, play etc should be separate. If we try to keep all of them together in one single test case, if one validation fails, none of the other methods will be validated.

File: cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        Then 'Firebolt' platform responds to '1st party app' with 'value for provide status'
+        When 1st party app invokes the 'Firebolt' API to 'get player status'
+        Then 'Firebolt' platform responds to '1st party app' with 'value for player status'
+        When 1st party app invokes the 'Firebolt' API to 'get broadcast status of player'
+        Then 'Firebolt' platform responds to '1st party app' with 'broadcast status of player'
+        When 1st party app invokes the 'Firebolt' API to 'stop player'
+        Then 'Firebolt' platform responds to '1st party app' with 'value for player stop'
+        When 1st party app invokes the 'Firebolt' API to 'destroy player'
+        Then 'Firebolt' platform responds to '1st party app' with 'value for player destroy'
+
```
Comment (neeradanelxsi) [inline cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature:38]: also, regarding the 3rd flow , check with kevin on the event name
also the difference in methods for streaming player and broadcast player,  mention that to kevin. Maybe it was a miss or it was intentional

File: cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    @initialization
+    Scenario: Launch FCA for 'IntegratedPlayer'
+        Given the environment has been set up for 'IntegratedPlayer' tests
+        And 3rd party 'certification' app is launched with 'inactive' state
+
+    @sdk @IntegratedPlayer
+    Scenario: Positive Scenario : Create and validate broadcast player instance
+        When 1st party app invokes the 'Firebolt' API to 'create broadcastPlayer instance'
+        Then 'Firebolt' platform responds to '1st party app' with 'broadcastPlayer instance'
+        When 1st party app invokes the 'Firebolt' API to 'get status of broadcastPlayer'
```
Comment (neeradanelxsi) [inline cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature:12]: also we can place TC first before creating a player . 

- So suppose if we call player.load, player.play without creating a broadcast/streaming player, will the platform throw error ?

or what does player.destroy do ? 
will it destroy the player ?  can you confirm that?

File: cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature [hunk 5]
```diff
@@ -0,0 +1,64 @@
+Feature: IntegratedPlayer
+
+    @initialization
+    Scenario: Launch FCA for 'IntegratedPlayer'
+        Given the environment has been set up for 'IntegratedPlayer' tests
+        And 3rd party 'certification' app is launched with 'inactive' state
+
+    @sdk @IntegratedPlayer
+    Scenario: Positive Scenario : Create and validate broadcast player instance
```
Comment (neeradanelxsi) [inline cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature:9]: we would need to write the tc in xumo and mf config module.

File: cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature (hunk 2 above)
Comment (Nandana-NNR) [reply cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature]: api and behavorial testcases seperated with player creation as a single test

File: cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature (hunk 5 above)
Comment (Nandana-NNR) [reply cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature]: ported to config module

File: cypress/support/constants/constants.js [hunk 6]
```diff
@@ -303,6 +303,7 @@ module.exports = {
   PLATFORM_NOT_SUPPORT_LOG: 'Platform does not support method',
   PLATFORM_NOT_TRIGGER_EVENT: 'Platform MUST not trigger event ',
   PLATFORM_TRIGGER_EVENT: 'Platform MUST trigger event ',
+  PLAYERID: 'playerId',
   PREREQUISITE_DATA: 'PreRequisiteData.json',
```
Comment (neeradanelxsi) [inline cypress/support/constants/constants.js:307]: not sure why we keep adding player in FCS. This has to be removed

Comment (Nandana-NNR) [reply cypress/support/constants/constants.js:307]: removed

**Pull request closed without merging**